src/components/structures/MatrixChat.js

```diff
@@ -249,6 +250,10 @@ module.exports = React.createClass({
                 register_hs_url: paramHs,
             });
         }
+
+        this._sessionStore = sessionStore;
```

Comment:
(and even if there is, isn't the point of a flux store that we can just pass the store object down to lower-level controllers (eg LoggedInView) rather than having a mahoosive `state` in the top-level component?